test/src/main/java/org/apache/accumulo/test/proxy/SimpleProxyBase.java

```diff
@@ -818,6 +950,58 @@ public void tableNotFound() throws Exception {
   }
 
   @Test
+  public void namespaceNotFound() throws Exception {
+    final String doesNotExist = "doesNotExists";
+    try {
+      client.deleteNamespace(creds, doesNotExist);
```

Comment:
If this thrift call does not throw a NNFE, it seems like the test will still pass?  If this is the case, could place an Assert.fail() after each thrift call thats supposed to throw.